libs/wire-subsystems/src/Wire/UserSubsystem.hs

```diff
@@ -115,6 +117,17 @@ getLocalUserProfile :: (Member UserSubsystem r) => Local UserId -> Sem r (Maybe
 getLocalUserProfile targetUser =
   listToMaybe <$> getLocalUserProfiles ((: []) <$> targetUser)
 
+getLocalUserAccountUnverified :: (Member UserSubsystem r) => Local UserId -> Sem r (Maybe UserAccount)
```

Comment:
is this a good name? or a good function? use `getAccountsBy` instead?

if you want to keep it, maybe call it `getLocalUserAccountIncludePending`?